.github/bot-action/main.py

```diff
@@ -0,0 +1,51 @@
+# TODO: improve this to support multiple commands
+import base64
+import os
+
+import httpx
+
+API_URL = os.environ["BOT_API_URL"]
+API_TOKEN = os.environ["API_SECRET"]
+
+
+mutation = """mutation AddReleaseComment($input: AddReleaseFileCommentInput!) {
+  addReleaseFileComment(input: $input)
+}"""
+
+
+release_info = None
+
+if os.environ["INPUT_STATUS"] == "OK":
+    changelog = base64.b64decode(os.environ["INPUT_CHANGELOG_BASE64"]).decode("utf-8")
+
+    release_info = {
+        "changeType": os.environ["INPUT_CHANGE_TYPE"],
+        "changelog": changelog,
+    }
+
+if tweet := os.environ.get("INPUT_TWEET"):
+    tweet = base64.b64decode(tweet).decode("utf-8")
+
+
+mutation_input = {
+    "prNumber": int(os.environ["INPUT_PR_NUMBER"]),
+    "status": os.environ["INPUT_STATUS"],
+    "releaseCardUrl": os.environ.get("INPUT_RELEASE_CARD_URL"),
+    "tweet": tweet,
+    "releaseInfo": release_info,
+}
```

Comment:
uh, I think I need to update the API to support this repo too

Reply:
Found the code, I see! I'll send you a PR updating that as well today. Should be easy!